core/string/ustring.h

```diff
@@ -520,7 +518,17 @@ class String {
 	String path_join(const String &p_file) const;
 	char32_t unicode_at(int p_idx) const;
 
+	void parse_utf32(const StrRange<char32_t> &p_cstr);
+
 	CharString ascii(bool p_allow_extended = false) const;
+	CharString latin1() const { return ascii(true); }
+	void parse_latin1(const StrRange<char> &p_cstr);
+	static String latin1(const CharString &p_string) {
+		String string;
+		string.parse_latin1(p_string);
+		return string;
+	}
```

Comment:
Come to think of it, all the other static from-encoding functions could just be inlined in the header, couldn't they? I'm not saying that's necessarily in the scope of this PR, but something to consider for the future.

Reply:
I agree, and I've been planning to do that. Small steps :)